dll/win32/kernel32/winnls/nls/ptg.nls

```diff
@@ -101,12 +101,12 @@ STRINGTABLE LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
   LOCALE_SDECIMAL ","
   LOCALE_SENGCOUNTRY "Portugal"
   LOCALE_SENGCURRNAME "Euro"
-  LOCALE_SENGLANGUAGE "Portuguese"
+  LOCALE_SENGLANGUAGE "Português"
   LOCALE_SGROUPING "3;0"
   LOCALE_SINTLSYMBOL "EUR"
   LOCALE_SISO3166CTRYNAME "PT"
   LOCALE_SISO639LANGNAME "pt"
-  LOCALE_SLANGUAGE "Portuguese (Portugal)"
+  LOCALE_SLANGUAGE "Português (Portugal)"
```

Comment:
This should be fixed separately for all languages, in the context of [CORE-18410](https://jira.reactos.org/browse/CORE-18410).

Please leave as is in this PR.